wradlib/io/xarray.py

```diff
@@ -1691,6 +1706,12 @@ def _merge_moments(self):
             ds = (ds.assign_coords({'azimuth': self.azimuth}).sortby('azimuth')
                   .pipe(_reindex_azimuth, self)
                   .drop('azimuth'))
+        else:
+            coords = mom.parent._get_coords().coords
```

Comment:
This is `self` instead of `mom.parent` since GAMIC has all moments already in one dataset.